packages/codegen-ui/lib/types/view/view.ts

```diff
@@ -18,14 +18,16 @@ import { ViewStyle } from './style';
 import { ColumnsMap } from './table';
 
 export interface StudioView {
-  appId?: string;
```

Comment:
One thing to note is that this interface is used to instantiate StudioView model in UI.

Reply:
I don't think it'll break UI but just a note

Reply:
Maybe sync w/ Kevin